build.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
-make -j T=open_source DEBUG=1 >log.txt 2>&1
+
+# use LANG environment variable, default to "en"
+LANG="${LANG:-en}"
+
+make -j T=open_source LANG=${LANG} DEBUG=1 >log.txt 2>&1
```

Comment:
It is unnecessary to specify variables on the make command-line if they are already in the environment.

```suggestion
make -j T=open_source DEBUG=1 >log.txt 2>&1
```